todo/src/components/Todo.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { addTodo, toggleTodo, deleteTodo } from "../actions";
+import './Todo.css';
+class Todo extends Component {
+  constructor() {
+    super();
+    this.state = {
+      todo: ""
+    };
+  }
+
+  handleChanges = event => {
+    this.setState({ [event.target.name]: event.target.value });
+  };
+
+  handleAddTodo = event => {
+    event.preventDefault();
+    this.props.addTodo(this.state.todo);
+  };
+
+  handleToggleTodo = index => {
+    this.props.toggleTodo(index);
+  };
```

Comment:
Why have a method for this when you're just taking the same parameter and passing it to your method on props?